frontend/src/scenes/actions/actionLogic.ts

```diff
@@ -106,17 +112,19 @@ export const actionLogic = kea<actionLogicType>([
             (action) => action?.steps?.some((step) => step.properties?.find((p) => p.type === 'cohort')) ?? false,
         ],
 
-        activityFilters: [
-            (s) => [s.action],
-            (action): ActivityFilters | null => {
-                return action?.id
-                    ? {
-                          scope: ActivityScope.ACTION,
-                          item_id: String(action.id),
-                      }
-                    : null
-            },
-        ],
+        // [SIDE_PANEL_CONTEXT_KEY]: [
```

Comment:
This probably wants to be added back in right? At least the activity scope if not the access controls

Reply:
I was trying to limit the across of access control for this first iteration. But yes you're correct so will look at making sure the activity scope is still set.